libs/xmlschemachecker/test/data/PACK.xsd

```diff
@@ -0,0 +1,1980 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could we pull this file from its original location?
https://github.com/Open-CMSIS-Pack/Open-CMSIS-Pack-Spec/tree/main/schema